index.js

```diff
@@ -22,15 +27,26 @@ const sessionStore = new MySQLStore({}, connection);
 
 // init server port
 //const port = 3000; 
-var PORT = envPort || 3000;
-var server = app.listen(PORT, function () {
-    console.log("Listening at port " + PORT + "...");
-});
 
 const options = {
-    useNewURLParser: true, 
-    useUnifiedTopology: true
-}
+    key: fs.readFileSync(path.join(__dirname, 'certs', 'server.key')),
+    cert: fs.readFileSync(path.join(__dirname, 'certs', 'server.crt')),
+};
+
+  
+//var PORT = envPort || 3000;
+// var server = app.listen(PORT, function () {
+//     console.log("Listening at port " + PORT + "...");
+// });
+
+https.createServer(options, app).listen(3000, () => {

```

Comment:
also please use the defined constants i already created like envPort

Reply:
same with the string in console.log the "use localhost:3000" pls change it so that the string also uses envPort

Reply:
![image](https://github.com/user-attachments/assets/13ba6149-e527-4594-b14b-aab93c7a71e5)

please define something like this uncomment it